jenkins_pipelines/environments/manager-mu-aws

```diff
@@ -28,7 +28,7 @@ node('sumaform-cucumber') {
     stage('Checkout pipeline') {
         checkout scm
     }
-    timeout(activity: false, time: 10, unit: 'HOURS') {
+    timeout(activity: false, time: 20, unit: 'HOURS') {
```

Comment:
```suggestion
    timeout(activity: false, time: 2, unit: 'HOURS') {
```

Reply:
We don't want to use activity timeout in cloud, as we don't want in PR tests.
Because we have some activities like mirroring content, or building content, that doesn't print feedback in the console output.